junit5/src/main/java/cz/xtf/junit5/extensions/helpers/PodLogItem.java

```diff
@@ -0,0 +1,60 @@
+package cz.xtf.junit5.extensions.helpers;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class PodLogItem {
```

Comment:
This messages you are abstracting are EAP messages. They follow some standards, however we should not expect just EAP logs.